extension/lib/addProductsToCart.js

```diff
@@ -2,7 +2,6 @@
 
 const ShopgateCartPipeline = require('./shopgate/CartExtensionPipeline')
 const { decorateError } = require('./shopgate/logDecorator')
-const ShopgateCartMessage = require('./shopgate/cart/Message')
```

Comment:
u removed this, I betcha this class is not needed anymore? If it does, I expect that implementation to be cleaned up too